### ISSUE TITLE
Can I jump to other "Views" in command mode?

### ISSUE THREAD

**Issue by llinfeng** [jupyterlab/jupyterlab#6925]
Is there an existing "method" that can shift the focus of the cell between two Views? For now, if I want to copy/manipulate cells in the other View, I need to reach to my mouse and click on such cell in the other view.

Ideally, something directional would be the best. In Vim, if I divide one window into 4 quadrants, I can use `ctrl + w` and then `h/j/k/l` to go to the desired quadrants. A demo goes below:
![Moving_cursors_among_panes](https://user-images.githubusercontent.com/5890407/62183231-615b9880-b327-11e9-8fc8-797c24940502.gif)

**Comment by jasongrout** [comment]
There are keyboard shortcuts to activate the next tab, those these are still a little awkward for switching to another tab pane without changing the tabs in that pane.

**Comment by jasongrout** [comment]
Leaving as open for exploring commands/keyboard shortcuts for switching between panes without changing the displayed tabs inside each pane.

**Comment by llinfeng** [comment]
@jasongrout Exactly, I now use the tab-switching shortcut while dividing the full window only by once, vertically. Further, when I need to go between two panes quickly, I shall close all other tabs and keep only the two. This is effectively not too far from the directional pane-jumping. Anyways, I also did not find a quick way to generate, say, vertical splits, without the mouse.
